tests/internal_conn/child/child.model.xml

```diff
@@ -0,0 +1,10 @@
+<models xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
Please add a `README.rst` to the `tests/internal_conn` directory describing the test.

Reply:
Done.